examples/benchmarks/nbody/java/nbody.java

```diff
@@ -0,0 +1,175 @@
+/* The Computer Language Benchmarks Game
```

Comment:
Did you take this directly from the language shootout? It looks different to the one in the warmup experiment. Not sure which one we should use...